fitting/fitScanData.py

```diff
@@ -50,7 +55,8 @@ def fit(self):
 
         random = r.TRandom3()
         random.SetSeed(0)
-        fitTF1 = r.TF1('myERF','500*TMath::Erf((TMath::Max([2],x)-[0])/(TMath::Sqrt(2)*[1]))+500',1,253)
+        fitTF1 = r.TF1('myERF','[3]*TMath::Erf((TMath::Max([2],x)-[0])/(TMath::Sqrt(2)*[1]))+[3]',1,253)
+        fitTF1.FixParameter(3, self.Nev / 2.)
```

Comment:
I would prefer we did not add to the list of fit parameters of this TF1.  Please instead use:

```
fitTF1 = r.TF1("myERF","%i*TMath::Erf((TMath::Max([2],x)-[0])/(TMath::Sqrt(2)*[1]))+%i"%(self.Nev / 2, self.Nev / 2.),1,253)
```

And remove line 59.